spec/features/users/dashboard/experienced_user_spec.rb

```diff
@@ -47,15 +56,25 @@
             expect(page).to have_css("#event-#{event.id}")
 
             within "#event-#{event.id}" do
+              # TODO: Once we have the Event endpoint returning the gym name,
+              #       we can add the `gym_name` attribute to the poro and
+              #       uncomment this line below.
+              # expect(page).to have_content(event.gym_name)
```

Comment:
Note to team - BE update in progress via this [issue](https://github.com/tvaroglu/spot_me_backend/issues/62)